conda_forge_tick/make_graph.py

```diff
@@ -57,13 +59,36 @@
 ]
 
 
+def get_deps_from_outputs_lut(
+    req_section: Iterable,
+    outputs_lut: typing.Dict[str, set],
+) -> set:
+    deps = set()
+    for req in req_section:
+        i = as_iterable(outputs_lut.get(req, req))
+        deps.update(i)
+    return deps
+
+
+def make_outputs_lut_from_graph(gx):
+    outputs_lut = defaultdict(set)
+    for node_name, node in gx.nodes.items():
+        for k in node.get("payload", {}).get("outputs_names", []):
+            if node_name != "pypy-meta":
+                outputs_lut[k].add(node_name)
+            elif k == "pypy":
```

Comment:
Is this correct? It seems that this will never be triggered.

Reply:
Yes. It is correct. The pypy-meta node (which is a feedtsock), has python, cffi and pypy as outputs:

https://github.com/conda-forge/pypy-meta-feedstock/blob/master/recipe/meta.yaml#L48

We only want to map it to pypy.